api/rest-api/endpoints/team-collaboration-endpoints/README.md

```diff
@@ -124,7 +124,7 @@ These methods apply to public channels only. Use `groups.*` methods for private
 
 | URL | Short Description | Details Page |
 | :--- | :--- | :--- |
-| `/api/v1/custom-sounds.list` | Retrieves a list of custom sounds. | ​[Link](https://app.gitbook.com/@rocket-chat/s/rocket-chat-developer/~/drafts/-MgSmy428hAP1znS6g3V/api/rest-api/endpoints/team-collaboration/custom-sounds/list)​ |
+| `/api/v1/custom-sounds.list` | Retrieves a list of custom sounds. | [Link](https://app.gitbook.com/@rocket-chat/s/rocket-chat-developer/~/drafts/-MgSmy428hAP1znS6g3V/api/rest-api/endpoints/team-collaboration/custom-sounds/list) |
```

Comment:
I noticed there were a heap of [zero-width space](https://en.wikipedia.org/wiki/Zero-width_space) characters dotted throughout the documentation. These are traditionally used to indicate word boundaries without the use of hyphenation, but where they were found would not have achieved that purpose (especially given they are in markdown). Any of the changes in this PR where there is no visual change is going to be the removal of the ZWSP's :)